### PR TITLE
Store index2

### PR DESCRIPTION
**Pull request by blsrofe**
Additions: 
Fix test for adding image to store path

User Story:

Concerns (code location):

Mentions to collaborators:

Notes(if applicable):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +35,3 @@ Site built with Ruby on Rails utilizing ActiveRecord on Postgresql. Site is prim
 ### Built With
 
  * Rails, Ruby, ActiveRecord, HTML, CSS, ERB, Postgresql, and Heroku.
```
Comment (mikeheft) [inline README.md:37]: Can we add versions that are used for at least the Ruby and Rails?

File: app/views/stores/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+<h1>Stores</h1>
+
+<div class="row">
   <% @stores.each do |store| %>
-    <%= link_to store.name, store_path(store.slug) %>
+  <div class="col s12 m6 l3">
+    <div class="card medium">
+      <div class="card-image">
+        <% if store.items.nil? %>
+          <%= image_tag "bath4.jpg" %>
```
Comment (mikeheft) [inline app/views/stores/index.html.erb:9]: Nice. That error was annoying :P

**Review by mikeheft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amhursh) [reply README.md:37]: Agreed I think this is something we can add.

File: app/views/stores/index.html.erb (hunk 2 above)
Comment (amhursh) [reply app/views/stores/index.html.erb:9]: This is a nice solution

**Pull request merged**